includes/Tables/NumberStoreListTable.php

```diff
@@ -218,12 +224,16 @@ public function get_numbers() {
 
 		if ( ! empty( $table_name ) ) {
 			if ( $search ) {
-				$query = $wpdb->prepare( "SELECT * FROM {$table_name} WHERE `id` = %d OR `order_id` = %d ORDER BY {$orderby} {$order}", $search, $search );
+				$results = $wpdb->get_results( 
+					$wpdb->prepare(
+						"SELECT * FROM `" . esc_sql( $table_name ) . "` WHERE `id` = %d OR `order_id` = %d ORDER BY " . esc_sql( $orderby ) . " " . esc_sql( $order ), 
+						$search, 
+						$search 
+					) 
+				);										
 			} else {
-				$query = "SELECT * FROM {$table_name} ORDER BY {$orderby} {$order}";
+				$results = $wpdb->get_results( "SELECT * FROM `" . esc_sql( $table_name ) . "` ORDER BY `" . esc_sql( $orderby ) . "` " . esc_sql( $order ) );
```

Comment:
No, the `esc_sql( $order )` does not require backticks because `$order` is expected to be a SQL keyword, such as ASC or DESC. Backticks are used for SQL identifiers (e.g., table or column names), not keywords.